content/5_Plat/RURQ.mdx

```diff
@@ -221,9 +221,166 @@ queries.
 	</Resource>
 </Resources>
 
-### Implementation
+### Solution - Range Updates & Sums
 
-<IncompleteSection />
+<Spoiler title="Solution">
+
+This question asks you to support the following types of queries. 
+
+-	Add a value to all elements within the range $[a,b]$.
+
+-	Set all values within the range $[a,b]$ to a certain value. 
+
+-	Find the sum of all values in range $[a,b]$.
+
+Consider the first two types of queries, a lazy tag will be created in each node of the tree for each type. In this solution, $lzAdd$ will represent the lazy tag for the range add query and $lzSet$ will represent the lazy tag for the range set query. 
```

Comment:
```suggestion
Consider the first two types of queries. A lazy tag will be created in each node of the tree for each type. In this solution, $lzAdd$ will represent the lazy tag for the range add query and $lzSet$ will represent the lazy tag for the range set query. 
```

Also, it might look better if you do something like $\texttt{lzAdd}$ instead of just $lzAdd$, but either one is fine with me